lib/cli.rb

```diff
@@ -197,6 +199,23 @@ def reject_test_names_option(parser)
           @reject_test_names = reject_test_names
         end
       end
+
+      def reject_report_section_option(parser)
+        parser.on('--reject-report-sections <reject_report_section>', Array,
+                  'List of section to reject from HTML results\n (use ' \
```

Comment:
This `\n` will not be a newline as it's single-quoted.

I also suggest to end the first literal with `\n`; it should be easier to read in this way. Perhaps you may use a heredoc, but I worry the syntax will be too complicated due to the block.